lib/src/string_extension.dart

```diff
@@ -1,3 +1,5 @@
+import 'package:characters/characters.dart';
```

Comment:
I mean, I can replace it `import 'package:flutter/material.dart';`, but some dependency is needed.

Reply:
So the point is, it does some extra magic to make sure that the split is actual characters. Since, as mentioned, in case of emojis, substring is splitting the emoji byte in malformed way, which is not intended.

The options are:
- Include `characters` into pubspec.yaml and import it.
- Or  just import `'package:flutter/material.dart';`, since it's included in there.

I decided to go the first way to make a more explicit and minimal dependency, rather than just whole materials.